2-api/models.md

```diff
@@ -1,72 +1,70 @@
 [Perspective API documentation](../README.md) > [API Reference Docs](README.md) > **Models and attributes**
 
-# Models and attributes
+# Attributes
 
-A model is a set of attributes (sometimes called "model attributes") on which a comment is scored on. For example, the Toxicity model contains attributes such as "Toxcity", "Profanity", "Fliratation", and more. [See all model types](#all-model-types). Models can be in-production or experimental.
+An attribute (sometimes called "model attributes" or "models") is a label on which a comment is scored. For example, the some attributes scored by Perspective API include as "Toxcity", "Profanity", "Fliratation", and more. [See all attribute types](#all-attribute-types). Attributes can be in-production or experimental.
```

Comment:
Remove "model attribute" for simplicity so it's just "sometimes called models" ?